src/Features/LanguageServer/Protocol/Features/CodeFixes/CodeFixService.cs

```diff
@@ -156,6 +156,9 @@ internal partial class CodeFixService : ICodeFixService
                 SortedDictionary<TextSpan, List<DiagnosticData>> spanToDiagnostics,
                 CancellationToken cancellationToken)
             {
+                // Ensure we yield here so the caller can continue on.
+                await AwaitExtensions.ConfigureAwait(Task.Yield(), false);
```

Comment:
we have a lot of usage of both Roslyn.Utilities and VSThreading in this file.  so i couldn't comment out the using for either easily.  So this was how i got past the ambiguity about which ConfigureAwait to call.